src/stan/variational/families/normal_fullrank.hpp

```diff
@@ -49,12 +49,15 @@ namespace stan {
 
     public:
       // Constructors
+
+      // Constructs a zero-filled variational distribution
       explicit normal_fullrank(size_t dimension) :
         dimension_(dimension) {
         mu_     = Eigen::VectorXd::Zero(dimension_);
-        L_chol_  = Eigen::MatrixXd::Identity(dimension_, dimension_);
+        L_chol_ = Eigen::MatrixXd::Zero(dimension_, dimension_);
```

Comment:
In the entropy calculation, why the loop to avoid zeroes.  Those can be super costly computationally in branch-point misprediction.  Shouldn't the diagonals be positive by construction and constraint?  And if they are not positive (the code only checks identity to zero, but there are also negative and not-a-number values), shouldn't the entropy explode?

Also, operators should start lines, not end lines as in the variable `result` for `entropy()`.